### ISSUE TITLE
how many maximum participant can be in single room in case of "multiparti conference"?

### ISSUE THREAD

**Issue by chiragmewada** [akumrao/mediaserver#2]

**Comment by akumrao** [comment]
Yes, Mediaserver supports webrtc.  Check the sample mediaserver/src/sfu/webRTCAppCheck.
16 Participants can Join a Room, which requires one Core Processor Power.
If you need bigger room. Then you can join the Room.

**Comment by chiragmewada** [comment]
Thanks for details. 

If I have 8 Core machine can it support 16 x 8 = 128 users? 

also how can I control audio and video stream of remote user to reduce bandwidth issue on client side?
